src/mon/FSCommands.h

```diff
@@ -87,4 +87,12 @@ class FileSystemCommandHandler : protected CommandHandler
     std::ostream &ss) = 0;
 };
 
+
+const std::string errmsg_for_unhealthy_mds = \
+  "MDS has one of two health warnings which could extend recovery: "
+  "MDS_TRIM or MDS_CACHE_OVERSIZED. MDS failover is not recommended "
+  "since it might cause unexpected file system unavailability. If "
+  "you wish to proceed, pass --yes-i-really-mean-it";
```

Comment:
```suggestion
static constexpr auto errmsg_for_unhealthy_mds = \
  "MDS has one of two health warnings which could extend recovery: "
  "MDS_TRIM or MDS_CACHE_OVERSIZED. MDS failover is not recommended "
  "since it might cause unexpected file system unavailability. If "
  "you wish to proceed, pass --yes-i-really-mean-it"sv;
```

Reply:
Fixed.

Reply:
Change has been pushed. PTAL.

Reply:
You forgot the "sv" suffix which makes it a string_view literal.